test_scripts/RC/OnRCStatus/029_notification_by_revoking_several_modules_by_policy.lua

```diff
@@ -52,9 +53,9 @@ runner.Step("Register RC application 1", common.registerRCApplication)
 runner.Step("Activate App 1", common.activateApp)
 runner.Step("Allocating module CLIMATE", alocateModule, { "CLIMATE" })
 runner.Step("Allocating module RADIO", alocateModule, { "RADIO" })
+runner.Step("Register application 2", common.registerApp, { 2 })
```

Comment:
**remove "Magic Number"** 

1. Add local variable
```
--[[ Local Variables ]]
local appSessionId2 = 2
```

2. Replace magic number to variable
2 = > appSessionId2
```suggestion
runner.Step("Register application 2", common.registerApp, { appSessionId2 })
```

Reply:
I wouldn't do this since we try to make less changes as possible in existing scripts.